### DISCUSSION TITLE
workflows > Is not empty

### DISCUSSION THREAD

**Discussion by IanMuller** [YetiForceCompany/YetiForceCRM#15007] [Feature improvement]
workflow condition - "Is not empty" are not working as expected. If I set it on UI type 10 field, even field is very much empty, the workflow will trigger anyway (with mentioned condition)

**Comment by rskrzypczak** [comment]
I checked it on the current version and I do not see any irregularities here.
Are you sure you haven't missed something?

By the way.. any further discussion on this topic requires first providing the version of the application and a specific example.
